ts/Extensions/Boost/BoostSeries.ts

```diff
@@ -690,6 +690,24 @@ function exitBoost(
     series: Series
 ): void {
     const boost = series.boost;
+    const chart = series.chart;
+    const chartBoost = chart.boost;
```

Comment:
Also, do we need to set `chart.boost` object as a new variable? New variable is always some extra bytes and in that case I think we can just use `chart.boost`.

Reply:
Thing is that we call the variable 3 times in a row so to me it makes sense from a performance perspective to store the reference to `chart.boost` 🤔 

I'm totally open to changing it but when is it appropriate to make/not make a new variable, @TorsteinHonsi ?

Reply:
I'm fine with the way it is. We save some bytes downstream when using a local variable. Regarding performance I think it is negligible in this case, since it's not a part of the code that runs thousands of times.